### PR TITLE
Adding support for building 32 bit boxes

### PR DESCRIPTION
**Pull request by rokaz**
Hi,

As discussed on Jabber, I send this PR for adding 32 bit boxes building support. There is also a fix for a problem with Virtualbox where the first virtual interface couldn't be brought up due to the absence of the cable signal, resulting in an impossibility to build the box.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,32 +19,42 @@ vagrant plugin install configure_networks
...
+*For 64 bit boxes*
 ```bash
-vagrant box add yunohost/jessie-stable https://build.yunohost.org/yunohost-jessie-stable.box
-vagrant box add yunohost/jessie-testing https://build.yunohost.org/yunohost-jessie-testing.box
-vagrant box add yunohost/jessie-unstable https://build.yunohost.org/yunohost-jessie-unstable.box
+vagrant box add yunohost/jessie-amd64-stable https://build.yunohost.org/yunohost-jessie-amd64-stable.box
+vagrant box add yunohost/jessie-amd64-testing https://build.yunohost.org/yunohost-jessie-amd64-testing.box
+vagrant box add yunohost/jessie-amd64-unstable https://build.yunohost.org/yunohost-jessie-amd64-unstable.box
+```
+*For 64 bit boxes*
```
Comment (M5oul) [inline README.md]: For 32 bit?

**Review by alexAubin — COMMENTED** [review]
Went around it pretty quickly. Don't know much about how to generate vagrant files, but that looks like relatively minor change / adaptation of pieces of code to 32 bits. Not tested but looks kind of good to me if other people can test it.

File: README.md (hunk 1 above)
Comment (rokaz) [reply README.md]: Yes it is a typo, I will correct this and add the commit to the PR

File: prebuild/README.md [hunk 2]
```diff
@@ -2,9 +2,16 @@
...
 
+### For the 64 bit box
+
 ```bash
 vagrant box add debian/contrib-jessie64
 ```
+### For the 32 bit box
+
+```bash
+vagrant box add bento/debian-8.6-i386
```
Comment (opi) [inline prebuild/README.md:13]: How far can we trust "bento" ?

Comment (rokaz) [reply prebuild/README.md:13]: This is what is recommended by the Vagrant documentation: https://www.vagrantup.com/docs/boxes.html (see Official Boxes paragraph).

Comment (opi) [reply prebuild/README.md:13]: oh, I didn't know ! thanks for that precious information; I'll test if "bento" boxes works well for building 64b images to

Comment (rokaz) [reply prebuild/README.md:13]: I also sent a mail to one guy in Debian who is involved in the Cloud team, who built the Vagrant base boxes you used for the 64 bit version, and proposed a .json for building a 32bit contrib-jessie box, if we prefer to have an official debian image. Though I couldn't test my .json, with their tools, you need a 64 bit system to build the base boxes.

**Review by zamentur — APPROVED** [review]

File: prebuild/README.md (hunk 2 above)
Comment (rokaz) [reply prebuild/README.md:13]: As an update, the Debian cloud team is not interested in maintaining 32 bit images of Debian for Vagrant, so I suggest to stick with the Bento images.

**Comment by alexAubin** [comment]
Trying to move forward on this : we have one approval from @zamentur. If there's no opposition I suggest to trust him and @rokaz and merge this in a few days.

**Pull request merged**